packages/siwx/vitest.config.ts

```diff
@@ -1,3 +1,7 @@
 import { defineProject } from 'vitest/config'
 
-export default defineProject({})
+export default defineProject({
+  test: {
+    environment: 'happy-dom'
```

Comment:
Any difference than using `jsdom` ? We usually use `jsdom` for all integration tests so curious.

Reply:
I had to use this because of lacking libraries on `jsdom` that are used in cryptography, like some missmatches of types in `Buffer` and `Uint8Array` .